### ISSUE TITLE
Pasted paragraphs are not divided into blocks

### ISSUE THREAD

**Issue by vierno** [facebook/draft-js#465]
If a blockRenderMap is not defined  with a 'p' element, pasted HTML with paragraph tags is not divided:

![jun-13-2016 18-35-20](https://cloud.githubusercontent.com/assets/681628/16023938/9cd260cc-3195-11e6-8d12-5d3b342fa008.gif)

Shouldn't the text be splitted into two 'unstyled' blocks?
Is this the intended behavior or a bug?

**Comment by J22Melody** [comment]
Similar issue: On my Mac, pasted paragraphs from Microsoft Word are not divided into blocks, but Pages works right.

**Comment by mitermayer** [comment]
Attached a PR that fixes this https://github.com/facebook/draft-js/pull/488

**Comment by mitermayer** [comment]
This issue has been fixed by https://github.com/facebook/draft-js/pull/828

**Comment by J22Melody** [comment]
@mitermayer Cool! I checkout draft-js 0.9.0, cherry-pick your commit and it works.

But I find a problem that if I set `blockRenderMap` like below:

```
unstyled: {
  element: 'p',
  aliasedElements: 'div',
},
```

and I try to paste some paragraph like `<p>123</p><p>234</p>` into my editor, it fails into a block again. 

I have noticed that in https://github.com/facebook/draft-js/blob/master/src/model/encoding/convertFromHTMLToContentBlocks.js#L173 it excludes `unstyledElement`. I wonder why is `unstyledElement` excluded in `BlockMapSupportedTags`? Does it cause my problem? Or I misunderstand something here?

Thanks a lot.
